src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentalPrice = 40;
+  const discount = 20;
+  const specialDiscount = 50;
+  const sumForAllDays = days * rentalPrice;
+
+  switch (true) {
+    case days < 3: {
+      return sumForAllDays;
+    }
+
+    case days < 7: {
+      return sumForAllDays - discount;
+    }
+
+    default:
+      return sumForAllDays - specialDiscount;
+  }
```

Comment:
i'm not rejecting it this time, but it's better to use plain if's instead of `switch(true` block, it's may be hard to understand

Reply:
got it